src/popover/__tests__/popover.test.js

```diff
@@ -254,45 +252,22 @@ describe('Popover', () => {
     const onClick = jest.fn();
     const content = <strong>Hello world</strong>;
 
-    const CustomComponent = (jest.fn(): any);
-    CustomComponent.mockReturnValue(<span>Hover Me</span>);
+    const CustomComponent = styled('div', {});
 
     wrapper = mount(
       <Popover isOpen content={content} onClick={onClick}>
         <CustomComponent />
       </Popover>,
     );
 
-    expect(CustomComponent).toHaveBeenCalled();
-    expect(CustomComponent).toHaveBeenCalledWith(
-      {
-        $ref: wrapper.instance().anchorRef,
-        onClick: wrapper.instance().onAnchorClick,
-        'aria-controls': null,
-        'aria-haspopup': 'true',
-        'aria-expanded': 'true',
-      },
-      {},
-    );
-    expect(CustomComponent).toHaveBeenLastCalledWith(
-      {
-        $ref: wrapper.instance().anchorRef,
-        onClick: wrapper.instance().onAnchorClick,
-        'aria-controls': 'bui-mock-id',
-        'aria-haspopup': 'true',
-        'aria-expanded': 'true',
-      },
-      {},
-    );
+    expect(wrapper).toMatchSnapshot();
```

Comment:
I think explicit test cases have more value here to make sure it functions as expected.